Website/sitecore/shell/client/Applications/WeBlog/Import.js

```diff
@@ -0,0 +1,138 @@
+﻿define(["sitecore"], function (Sitecore) {
+    var WordPressImport = Sitecore.Definitions.App.extend({
+        initialized: function () {
+            this.tabManager.showSelectFile(this);
```

Comment:
We should use consistent casing with the component IDs. May as well use a capitol for the first letter.